src/classes/RecipeBox.js

```diff
@@ -23,19 +24,26 @@ class RecipeBox {
     })
     return matchingRecipes;
   }
-  findRecipeByIngredient(theUserInput) {
-    let matchingIngredientRecipes = [];
-    ingredientData.forEach((ingredient) => {
-      if(ingredient.name.includes(theUserInput)) {
-        matchingRecipes.push(recipe)
+  findIngredientById(theUserInput) {
```

Comment:
I wonder if we could talk through this function, again, please and thank you?

Reply:
Of course!